### PR TITLE
Docs/update build guide for jan 31 2021 version/4

### LINKED ISSUE #4: Update build guide for January 31 2021 version

### PR DESCRIPTION
**Pull request by capsulecorplab**
closes #4

### CODE REVIEW

**Comment by capsulecorplab** [comment]
Converting PR to `Draft`, until STL files for the m12 controller body get updated to v3 in https://github.com/tetrabiodistributed/papr-print-files/pull/72

File: content/en/build-guide/m12-v3.adoc [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+This build differs from the v1 version in that it uses a custom-built circuitboard to remove many of the difficult steps of the build, add a battery charge indicator, and a separate power port for use in a car.  If you cannot get the custom board (found in this repository: https://github.com/tetrabiodistributed/PAPRA-PCB), then please refer to the v1 version of this guide.
+
+== Parts Required
+
+This build will require printed parts, purchased parts, and several tools.  Here is a picture of all the components used in this build:
+
+[link=https://photos.smugmug.com/Tetra-Testing/Tetra-PAPRa-Build-Party-31-Jan-2021/i-7W25d6J/0/ca9a585b/5K/_MMR0088-5K.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/Tetra-PAPRa-Build-Party-31-Jan-2021/i-7W25d6J/0/ca9a585b/L/_MMR0088-L.jpg[]
+
+This is the assembly diagram of the controller itself:
```
Comment (mmroden) [inline content/en/build-guide/m12-v3.adoc]: remove

File: content/en/build-guide/m12-v3.adoc [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+The assembly diagram for the board can be found here:
```
Comment (mmroden) [inline content/en/build-guide/m12-v3.adoc]: remove

File: content/en/build-guide/mask-v3.adoc [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+== Parts Required
+
+This photo shows all the parts laid out for building this component:
+
+
+[link=https://photos.smugmug.com/Tetra-Testing/Tetra-PAPRa-Build-Party-31-Jan-2021/i-kfLSx45/0/2e89f579/5K/IMG_5115-5K.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/Tetra-PAPRa-Build-Party-31-Jan-2021/i-kfLSx45/0/2e89f579/L/IMG_5115-L.jpg[]
+
+The assembly diagram can be found here:
```
Comment (mmroden) [inline content/en/build-guide/mask-v3.adoc:21]: remove

**Review by mmroden — APPROVED** [review]

**Pull request merged**